app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -0,0 +1,39 @@
+.gem-c-intervention {
+  @include govuk-text-colour;
+  @include govuk-responsive-padding(4);
+  @include govuk-responsive-margin(8, "bottom");
+
+  clear: both;
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+
+  .govuk-govspeak {
+    p:last-child {
+      margin-bottom: 0;
+    }
+  }
```

Comment:
I don't think that we should override the `govuk-govspeak` styles like this - it'd be better to add another class (example:`gem-c-intervention-govspeak`) and then use that to remove the margin. This way it's clearer where the styles are coming from.

Another thought is that I can't find where `govuk-govspeak` is used in the component - so is this still needed?

Reply:
None of the `govuk-` components are used at this point, so removing in the css